app/controllers/dailies_controller.rb

```diff
@@ -2,6 +2,7 @@ class DailiesController < ApplicationController
 
   def index
     @dailies = current_user.dailies
+    @chart_data = @dailies.map { |daily| daily.table_row}.reduce(:+)
```

Comment:
yup, i'd encourage you to refactor this and move it to a model method in "Daily" potentially. your controller shouldn't be performing this logic 👍